api/specs/common/schemas/scripts/remove_definitions.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+import yaml
+
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+source_file_name = sys.argv[1]
+target_file_name = sys.argv[2]
+file_source_path = DIR_PATH + f"/../{source_file_name}"
+file_target_path = DIR_PATH + f"/../{target_file_name}"
+
+with open(file_source_path, "r") as stream:
```

Comment:
also wondering. In case the definitions are equivalent to what we have I would keep them instead of having to do many manuel changes.

Reply:
This is used automatically in Make file when generating  node-meta-v0.0.1-**pydantic-converted-clean**.yaml

Reply:
so you mean it is necessary so that the startup of webserver does not complain? in that case that is ok. If it is equivalent then I would just go for the newest version. In the end the idea is to remove the old parts.